### DISCUSSION TITLE
Python language not detected in Django app repository

### DISCUSSION THREAD

**Discussion by WMRamadan** [github-linguist/linguist#6082] [Classification]
The following repository only detect Javascript/CSS/HTML/SCSS languages but does not seem to detect python. A .gitattributes file was added and all override parameters tried. At one point the repository did detect the Python language but then went back and Python language was gone.

https://github.com/WMRamadan/django-product-review-app

**Comment by lildude** [comment]
> but does not seem to detect python.

It does now, and it's regardless of your `.gitattributes` file as it's not having the effect you think it is 😁 Maybe you didn't wait long enough for the background analysis job to run?
